src/machine/machine-dbus.c

```diff
@@ -628,11 +628,17 @@ int bus_machine_method_open_shell(sd_bus_message *message, void *userdata, sd_bu
         if (!strv_env_is_valid(env))
                 return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "Invalid environment assignments");
 
+        const char *details[7] = {
```

Comment:
I'd prefer to leave the array size out, and have NULL as an explicit terminator below. That "7" is a bit surprising and having a terminator makes it less likely to be messed up on updates.

Reply:
Thanks, good idea.